code/modules/admin/sql_message_system.dm

```diff
@@ -255,9 +255,9 @@
 					notedata += data
 		output += "<h2><center>[target_ckey]</center></h2><center>"
 		if(!linkless)
+			output += " <a href='?_src_=holder;addnote=[target_ckey]'>\[Add note\]</a>"
 			output += "<a href='?_src_=holder;addmessage=[target_ckey]'>\[Add message\]</a>"
```

Comment:
Change the spacing here, the space in the now first line should be moved to this one.